src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/tests/test_aks_commands.py

```diff
@@ -8,12 +8,11 @@
 import unittest
 
 from azure.cli.testsdk import (
-    ResourceGroupPreparer, RoleBasedServicePrincipalPreparer, LiveScenarioTest)
+    ResourceGroupPreparer, RoleBasedServicePrincipalPreparer, ScenarioTest)
 
 # flake8: noqa
 
-# TODO: Convert back to ScenarioTest and re-record when issue #5141 is resolved
-class AzureKubernetesServiceScenarioTest(LiveScenarioTest):
+class AzureKubernetesServiceScenarioTest(ScenarioTest):
```

Comment:
What was the issue here?

Reply:
I'm not sure honestly. I wasn't able to reproduce the errors you saw when running `--live` so eventually I just re-recorded these two.